### ISSUE TITLE
Modal popups don't disappear after a successful use of the gw2 launcher.

### ISSUE THREAD

**Issue by mwrich4** [TheCheatsrichter/Gw2_Launchbuddy#144]
After finding and looking at the location where you keep the 1.dat etc, I finally deduced the necessary order of steps to set up your program for my accounts.  The boxes say they will disappear after a good session, but they didn't.  I have 5 accounts and not once did the box auto clear, even though the setup worked properly.

**Comment by TheCheatsrichter** [comment]
Launchbuddy actually listens for modules loaded by the gameclient.
So if for some reason the modules don't get loaded at all or are out of order these things can happen.

I am personally not able to reproduce this issue.
Could you provide additional information about your systems OS and hardware?

**Comment by mwrich4** [comment]
I'm sorry, I didn't give much useful background for investigation.

I'm using Windows 7 Pro, with an i7, 32GB RAM and an SSD for the game files.

LaunchBuddy is running with the 'Set Administrator' property.  Both gw2 and LB are being launched from my SSD at "S:\\Guild Wars 2\\"

I was in the process of trying to discover how to set up LB to launch individual gw2 accounts.  I had made an entry 'Main' and was trying to get the login credentials set up using Set Loginfile.  I would click Set Loginfile, the client login screen would open, I'd enter my user/pass, click run and then close that instance of gw2 when the character selection screen would appear.  When I looked back at LB, there was still a popup saying that it would autodetect the login file and then close itself.  It did not close.  The next window said the loginfile could not be detected and warned me.  I later learned that the login file was being generated and copied to the safe area properly.  Later use of starting 'Main' worked properly.

**Comment by mwrich4** [comment]
Perhaps this is residue from not approaching the initial set-up properly when I first started using LB.  Since then, I've had the last updates caused by the last 2 'patch' Tuesdays handled properly.  

LB seems happier if I DON'T update the gw2 files with a normal launch of gw2 before using LB.  I just start LB after a patch, and the updating seems to happen smoothly and LB updates all of the login.DAT files.  Perhaps my issue was caused by starting GW2, applying updates and closing like I had to do before with old LB (1.0.5).

I'll keep watching.
